### ISSUE TITLE
Stale data

### ISSUE THREAD

**Issue by tfalamu** [Mastersam07/ncovid-19-api#7]
Data seems to be stale again. Api results do not match ncdc data

**Comment by Mastersam07** [comment]
Scrapers working fine. As at when the refresh happened, the data seems not to have been updated. I want to avoid manual updates. However, the next refresh of data is by 3pm. Clock has also been reset. Refresh would be every 6hours starting from 3pm. This should fix changes to the data mid day.

**Comment by tfalamu** [comment]
Okay great

**Comment by tfalamu** [comment]
The work I'm doing can be found here: https://github.com/vague369/c19-android-widget. If you'd like to add it to your showcase

**Comment by tfalamu** [comment]
Still waiting on the api update though

**Comment by Mastersam07** [comment]
Yes. I raised up a new issue announcing some changes. I'm almost done. Should combat all the changes to structure hence API still gets functional #8

**Comment by Mastersam07** [comment]
Making some final updates.

**Comment by Mastersam07** [comment]
Fixed this and #8
